derex/runner/cli.py

```diff
@@ -178,9 +186,26 @@ def build_final(ctx, project):
 @click.pass_obj
 @click.pass_context
 @ensure_project
-def build_final_refresh(ctx, project):
+def build_final_refresh(ctx, project: Project):
     """Also pull base docker image before starting building"""
     from derex.runner.docker import pull_images
 
     pull_images([project.base_image, project.final_base_image])
     ctx.forward(build_final)
+
+
+@derex.command()
+@click.argument(
+    "runmode",
+    type=click.Choice(ProjectRunMode.__members__),
+    required=False,
+    callback=lambda _, __, value: value and ProjectRunMode[value],
+)
+@click.pass_obj
+@ensure_project
+def runmode(project: Project, runmode: Optional[ProjectRunMode]):
+    """Get/set project runmode (debug/production)"""
+    if runmode is None:
+        click.echo(project.runmode.name)
+    else:
+        project.runmode = runmode
```

Comment:
Maybe we can add some feedback about what was done (e.g. Runmode changed from "production" to "debug").

Reply:
Agreed. A small usability improvement is not much, but many of them can go a long way to make the whole project feel more ergonomic. Implementing.